app/views/chapters/manage.html.erb

```diff
@@ -37,10 +37,11 @@
             <% if @work.pseuds.size > 1 && chapter.pseuds.size > 1 && current_user.is_author_of?(chapter) %>
               <li>
                 <%= link_to ts("Remove Me As Chapter Co-Creator"),
```

Comment:
This has to be a `link_to`, because this element is already within a form for `update_positions`. There is already a `link_to` for Delete Chapter here so I left it be.